### DISCUSSION TITLE
How to resolve editor warnings: Because this call is not awaited

### DISCUSSION THREAD

**Discussion by tpnet** [KyryloKuzyk/PrimeTween#105] [FAQ] [answered]
When I write the following code in the editor,  code like this:
```
    private async UniTask TestUniTask()
    {
        var seq = Sequence.Create();
        if (!mBtnBroken.IsActive())
        {
            mBtnBroken.targetGraphic.color = ColorUtils.ClearWhite;
            mBtnBroken.Active(true);
            seq.Group(Tween.Alpha(mBtnBroken.targetGraphic, 1f, 1f));
        }
        seq.Group(Tween.Alpha(mBtnCd.targetGraphic, 0f, 1f));
        await seq;
    }
```

The Editor will display a warning： 
> Because this call is not awaited, execution of the current method continues before the call is completed. Consider applying the 'await' operator to the result of the call.

How to solve this warning?

**Comment by KyryloKuzyk** [comment, marked as answer]
There are multiple ways to suppress the warning and the easiest one I found is this one: https://stackoverflow.com/a/47790150/1951038

This warning appears because Tween and Sequence are both awaitable types and C# thinks that a user may have forgotten to call await in async method. But now I think whether it was a design mistake on my part to make Tween/Sequence awaitable types instead of introducing a separate `ToAwaitable()` method that a user should call when they want to await an animation.

**Reply by tpnet** [reply]
`_ = Tween` is work, I think it might be better to keep the current design.

**Reply by mwp007** [reply]
I do
```
_ = Tween

```

as well. Would stick to it too.
